src/NodeApi/Native/JSNativeApi.Interop.cs

```diff
@@ -4,15 +4,32 @@
 // Definitions from Node.JS js_native_api.h and js_native_api_types.h
 
 using System;
+using System.Diagnostics;
 using System.Runtime.CompilerServices;
 using System.Runtime.InteropServices;
 using System.Security;
 using System.Text;
+using Microsoft.JavaScript.NodeApi.Runtime;
 
 namespace Microsoft.JavaScript.NodeApi;
 
 public static partial class JSNativeApi
 {
+    // TODO: When complete, JSRuntime will mostly or fully replace the JSNativeApi class.
+    // TODO: Move this to a thread-local or attach it to the thread-local
+    // JSValueScope or JSRuntimeContext.
```

Comment:
My vote is for attaching it to `JSValueScope` and/or `JSRuntimeContext`. It should help associating JS values/objects with the correct runtime regardless of the thread.

Reply:
Done as (non-static) `JSValueScope.Runtime` and `static JSValueScope.CurrentRuntime => JSValueScope.Current.Runtime`.